### PR TITLE
OPENTOK-36018 - Adding E2E tests using WebDriver.io

### PR DESCRIPTION
**Pull request by aullman**
Adding webdriver.io and some tests for Basic Video Chat. The tests run in Chrome, Firefox, Safari and Internet Explorer 11 (using Sauce Labs).

OPENTOK-36018

### CODE REVIEW

**Comment by aullman** [comment]
failed errorShot

**Comment by aullman** [comment]
failed errorShot

**Comment by aullman** [comment]
chrome stable failed errorShot

**Comment by aullman** [comment]
chrome stable failed errorShot

**Comment by aullman** [comment]
firefox stable failed errorShot

**Comment by aullman** [comment]
firefox stable failed errorShot

**Comment by aullman** [comment]
safari stable failed errorShot

**Comment by aullman** [comment]
firefox stable failed errorShot

**Comment by aullman** [comment]
firefox stable failed errorShot

**Comment by aullman** [comment]
chrome stable failed errorShot

**Comment by aullman** [comment]
chrome stable failed errorShot

**Comment by aullman** [comment]
safari stable failed errorShot

**Comment by aullman** [comment]
chrome stable failed errorShot

**Comment by aullman** [comment]
chrome stable failed errorShot

**Comment by aullman** [comment]
firefox stable failed errorShot

**Comment by aullman** [comment]
firefox stable failed errorShot

**Comment by aullman** [comment]
safari stable failed errorShot

**Comment by aullman** [comment]
safari stable failed errorShot

**Comment by aullman** [comment]
Got Safari working 🎉

**Comment by aullman** [comment]
W00t IE is working too!

**Comment by aullman** [comment]
ie 11 failed errorShot

File: uploadErrorShots.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# Upload error images to imgur and post them as a github comment on the PR
+if [[ "$TRAVIS_PULL_REQUEST" != "false" && -d ./errorShots ]] ; then
+  curl https://raw.githubusercontent.com/tremby/imgur.sh/master/imgur.sh > imgur.sh
+  chmod u+x imgur.sh
+
+  for file in `ls errorShots/`; do
+    echo "Uploading: $file"
+    output=`./imgur.sh errorShots/$file`
+    echo $output
+    imgURL=`echo $output | grep .png`
```
Comment (aiham) [inline uploadErrorShots.sh]: escape slash

File: wdio.conf.js [hunk 2]
```diff
@@ -0,0 +1,302 @@
+let config = {
```
Comment (aiham) [inline wdio.conf.js]: const

**Comment by aullman** [comment]
ie 11 failed errorShot

**Comment by aullman** [comment]
ie 11 failed errorShot

**Pull request merged**